x-pack/plugins/security_solution/public/common/translations.ts

```diff
@@ -30,19 +33,22 @@ export const UPDATE_ALERT_STATUS_FAILED_DETAILED = (updated: number, conflicts:
          because { conflicts, plural, =1 {it was} other {they were}} already being modified.`,
   });
 
-export const UPGRADE_AGENT_FOR_RESPONDER = (_agentType: ResponseActionAgentType) => {
-  const agentType = snakeCaseToCamelCase(_agentType);
+export const UPGRADE_AGENT_FOR_RESPONDER = (
+  _agentType: ResponseActionAgentType,
+  command: ConsoleResponseActionCommands
+) => {
+  const agentType = getAgentTypeName(_agentType);
 
   if (_agentType === 'endpoint') {
-    return i18n.translate('xpack.securitySolution.endpoint.actions.disabledResponder.tooltip', {
-      defaultMessage: `The current version of the {agentType} Agent does not support this feature. Upgrade your Agent through Fleet to use this feature and new response actions such as killing and suspending processes.`,
-      values: { agentType },
+    return i18n.translate('xpack.securitySolution.endpoint.actions.unsupported.message', {
+      defaultMessage: `The current version of the {agentType} Agent does not support {command}. Upgrade your Elastic Agent through Fleet to the latest version to enable this response action.`,
```

Comment:
Maybe we should say `Upgrade your Endpoint Agent...` instead so that it is consistent with the previous sentence? @paul-tavares @caitlinbetz

Reply:
I still think there are different messages for different scenarios. For Elastic Agent/Endpoint, this is good. For external EDR, I don't think it correctly applies to them.

I'm ok leaving it as is until we can get proper messaging in place - would just ask that you track it under an issue so that we don't forget to revisit it.